src/js/utils/html-utils.js

```diff
@@ -6,58 +6,40 @@
  */
 (function(define) {
     'use strict';
-    define(['underscore', './string-utils.js'], function(_, StringUtils) {
+    define(['underscore', 'edx-ui-toolkit/js/utils/string-utils'], function(_, StringUtils) {
         var escape, interpolateHtml, HTML, template;
 
         /**
-         * Creates an HTML snippet.
-         *
-         * The intention of an HTML snippet is to capture a string that
-         * is known to contain HTML that has been safely escaped.
-         * As an example, this allows interpolate to understand that
-         * it does not need to further escape this HTML.
-         *
-         * @param {string} htmlString The string of HTML.
-         * @constructor
-         */
-        function HtmlSnippet(htmlString) {
-            this.text = htmlString;
-        }
-        HtmlSnippet.prototype.valueOf = function() {
-            return this.text;
-        };
-        HtmlSnippet.prototype.toString = function() {
-            return this.text;
-        };
-
-        /**
          * Helper function to create an HTML snippet.
          *
          * @param {string} htmlString The string of HTML.
-         * @returns {HtmlSnippet} An HTML snippet that can be safely rendered.
+         * @returns {string} An string flagged as an HTML snippet.
          * @constructor
          */
         HTML = function(htmlString) {
-            return new HtmlSnippet(htmlString);
+            var htmlSnippet = new String(htmlString); // jshint ignore:line
+            htmlSnippet.isHtmlSnippet = true;
+            return htmlSnippet;
         };
 
         /**
          * Returns HTML that is appropriately escaped.
          *
-         * If the input is provided as a string, then an HtmlSnippet
-         * is returned with the original text escaped. If the input
-         * is already an HtmlSnippet then it is known to be already
-         * escaped and so it is just returned directly.
+         * If the input is provided as a string, then a new string
+         * is returned with the original text escaped. In addition,
+         * the string is flagged as being an HTML snippet. If the input
```

Comment:
Fix comment.  This is not an HTML Snippet.  You may want to provide a sample of wrapping with HTML() if they need a snippet.